module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -658,6 +679,60 @@ namespace module::input {
                             }
                         }
                         else {
+                            /************************************************
+                             *       Torso CoM Position in World (rMWw)     *
+                             ************************************************/
+                            bool update_done = false;
+
+                            for (auto& side : {BodySide::LEFT, BodySide::RIGHT}) {
```

Comment:
There are r-values, aren't they?
```suggestion
                            for (auto&& side : {BodySide::LEFT, BodySide::RIGHT}) {
```

Reply:
Should probably be `const auto&`?

Reply:
sure